src/SystemCommands-RefactoringSupport-Tests/SycMethodNameEditorTest.class.st

```diff
@@ -177,9 +201,9 @@ SycMethodNameEditorTest >> testEditParameterShouldHaveGhostText [
 ]
 
 { #category : #'tests - initial state' }
-SycMethodNameEditorTest >> testInitialArgumentListHasNoSelection [
+SycMethodNameEditorTest >> testInitialArgumentListAutoSelection [
 	
-	self assert: methodNameEditor argumentsList selection isEmpty
+	self assert: methodNameEditor argumentsList selection selectedIndexes equals: #(1)
```

Comment:
Why not testing `isNotEmpty` ?
Unless you really care about the first one.

What happens in the list is empty?

Reply:
This is because now the first argument is selected (if it exists) by default, before it gave an error that when pressing Up or Down button because it did not refer to any selected element